### PR TITLE
created the select component

### PR DESCRIPTION
**Pull request by iamavs09**
created the select component

### CODE REVIEW

File: src/component/select/select.container.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const PFSelect = ({label,options,onChange,value,...props}) => {
```
Comment (imvedanshmehra) [inline src/component/select/select.container.jsx:1]: Please make the label prop optional

File: src/component/select/select.container.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      <div className="lg:col-span-4">
+      <label className="font-semibold">{label}:</label>
+      <select
+        className="form-select form-input mt-2 w-full py-2 px-3 h-10 bg-transparent dark:bg-slate-900 dark:text-slate-200 rounded outline-none border border-gray-200 focus:border-indigo-600 dark:border-gray-800 dark:focus:border-indigo-600 focus:ring-0"
+        value={value} 
+        onChange={handleChange} 
+        {...props}
+      >
+        {options?.map((option, index) => (
+          <option key={index} value={option.value}>
```
Comment (imvedanshmehra) [inline src/component/select/select.container.jsx:17]: don't use index as key, use option.value instead

File: src/component/select/select.container.jsx (hunk 1 above)
Comment (iamavs09) [reply src/component/select/select.container.jsx]: done

File: src/component/select/select.container.jsx (hunk 2 above)
Comment (iamavs09) [reply src/component/select/select.container.jsx]: done

**Pull request merged**